aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +270,53 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Where to present a form
+
+There are several placement options when presenting a form, but when should a particular 
+placement be chosen and why? On a dedicated page? Within a layer? If within a layer, which 
+type?
+
+<FormPresentationDecisionTree />
+
+## Forms in layers vs. full page 
+
+A full-page form is used to display information free from interuption.
+
+**Best practices for full-page forms:**
+- When using a full-page form use a single-column form layout to create a clear focused path to complete for the user.
+- When using a full-page form avoid using Multi-column layouts not to cause users to skip important fields, input data in the wrong field, or simply hinder efficiency which leads to frustration or even abandonment.
+- When using a single form-page use no more than 10 form fields in the same viewport so that users are not intimidated by seeing a long page filled with form fields and sections.
+
+**Best practices for forms in layers:**
```

Comment:
In my opinion, these are kinda stated above so I would take them out unless we kinda want them to restate what the diagram says but I don't think it would be bad if we removed

Reply:
Agree to removing.

Reply:
To me the "forms in layer vs full page" is confusing because it feels redundant with the decision tree.

Reply:
agree